docs/docs/installation/upgrade.md

```diff
@@ -24,7 +24,7 @@ If you installed your Keptn instance from the Manifest,
 additional steps are required to use the Helm Chart to upgrade.
 Contact us on Slack for assistance.
 
-## Upgrade to v1beta1 version
+## Upgrade to v1 version
```

Comment:
are you sure this is correct for the upgrade?

Reply:
yes, adapted the page, since v1 and v1beta1 are fully compatible